modules/unsupported/mbtiles/src/test/java/org/geotools/mbtiles/MBTilesFeatureSourceTest.java

```diff
@@ -293,6 +294,35 @@ public void testReadOutsideWorld() throws Exception {
         assertThat(rangesRead, Matchers.empty());
     }
 
+    @Test
+    public void testGetBBOXQueryALL() throws Exception {
+        File file =
+                URLs.urlToFile(MBTilesFileVectorTileTest.class.getResource("madagascar.mbtiles"));
+        this.store = new MBTilesDataStore(new MBTilesFile(file));
+        ReferencedEnvelope envelope = store.getFeatureSource("water").getBounds();
+        assertNotNull(envelope);
+        assertEquals(
+                new ReferencedEnvelope(
```

Comment:
Was a bit surprised but indeed the bbox in the medatata is the whole world:

![image](https://user-images.githubusercontent.com/325433/79772403-55d65480-8330-11ea-96f1-38bbe0f82c59.png)

So looking good.